modules/nw-configure-sysctl-interface-sriov-network-bonded.adoc

```diff
@@ -0,0 +1,204 @@
+// Module included in the following assemblies:
+//
+//networking/hardware_networks/configuring-sriov-device.adoc
+
+:_content-type: PROCEDURE
+[id="configure-sysctl-on-bonded-sriov-network_{context}"]
+= Configure sysctl on a bonded SR-IOV network 
+
+As a cluster administrator, set the interface specific sysctls on a bonded interface created from two SR-IOV interfaces by adding the tuning configuration to the optional `metaPlugins` parameter of the bond network attachment definition.
+
+
+[NOTE]
+====
+Do not edit `NetworkAttachmentDefinition` custom resources that the SR-IOV Network Operator manages. Doing so might disrupt network traffic on your additional network.
+====
+
+To change all interface-level network sysctls create an additional SR-IOV network attachment with the Container Network Interface (CNI) tuning plug-in by using the following procedure.
+
+.Prerequisites
+
+* Install the {product-title} CLI (oc).
+* Log in to the {product-title} cluster as a user with cluster-admin privileges.
+
+.Procedure
+
+. Create a bond network attachment definition and insert the `metaPlugins` configuration, as in the following example CR. Save the YAML as the file `sriov-network-interface-sysctl-all.yaml`.
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+    kind: NetworkAttachmentDefinition
+    metadata:
+      name: bond-net1
+      namespace: demo
+    spec:
+      config: '{
+      "type": "bond", <1>
+      "cniVersion": "0.3.1",
+      "name": "bond-net1",
+      "ifname: "bond0" <2>
+      "mode": "active-backup", <3>
+      "failOverMac": 1, <4>
+      "linksInContainer": true, <5>
+      "miimon": "100",
+      "mtu": 1500,
+      "links": [ <6>
+            {"name": "net1"},
+            {"name": "net2"}
+        ],
+      "ipam": { , <7> 
```

Comment:
In pod example you configure ip addresses manually, in that case you can simple set ipam to static:

```
"ipam":{
        "type":"static"
      }
```